### DISCUSSION TITLE
Interpretations of output for Amino Acids

### DISCUSSION THREAD

**Discussion by mjos19** [stjude/CICERO#114] [Q&A] [answered]
Specifically asking about the output of Sv_AA and Sv_interstitial_AA columns in the final_fusions.txt

Is the Sv_AA column in reference to the amino acid sequence of the whole fusion protein? or just the amino acid sequence spanning the structural variation? 

And what does the ",,," output" of the Sv_intersitial_AA column represent?

Would love help on this! Thank you

**Comment by mnedmonson** [comment, marked as answer]
It is just the amino acid sequence spanning the breakpoint, and is based on translation of CICERO's nucleotide contig sequence.  The interstitial field represents amino acid sequence at the breakpoint that was not aligned to either gene.  Some fields in the output are comma-delimited lists with each entry representing the result for a different combination of isoforms for genes A and B.  In this case it appears that there were four isoform combinations, but none of them had data for this field.
